libs/ui/src/table.tsx

```diff
@@ -17,8 +18,12 @@ export const Table = styled.table<TableProps>`
   & th,
   & td {
     border-bottom: 1px solid ${borderColor};
-    padding: ${({ condensed }) =>
-      condensed ? '0.15rem 0.25rem' : '0.25rem 0.5rem'};
+    padding: ${({ condensed, expanded }) =>
+      condensed
+        ? '0.15rem 0.25rem'
+        : expanded
+        ? '0.25rem 1rem'
+        : '0.25rem 0.5rem'};
```

Comment:
I'm not totally sure what the context for the expanded/condensed tables are, but it might be nice to keep the horizontal padding equal on all styles so that the text always lines up with other tables on the page, regardless of style. But this might not be a real concern, so feel free to ignore

Reply:
Yeah, this was a contrived example for a screenshot. At present don't use different tables on the same screen… and I don't think we will. 🤞